### ISSUE TITLE
Switching between SD 1.5 and SD XL base models?

### ISSUE THREAD

**Issue by olaxe** [ArtVentureX/sd-webui-agent-scheduler#111]
Hello,

With the increasing number of models using SD XL as base model, I need to switch the base model based on the target model.

My default loading base model is still 1.5 until more XL models will be there.
I found a hack to switch on SD XL base model by asking a text generation of the SD XL with steps = -1, so the task failed but the model remains loaded and I just to add another task with the SD XL target model I need.

I would like to know if there is a better way of doing it and do you plan to manage the switch between base models?

Thanks in advance,

**Comment by artventuredev** [comment]
Sorry if I get you wrong, but I haven't try SDXL on SD webui (I use comfy most of the time). Are you trying to switch the model between queued tasks?

**Comment by olaxe** [comment]
yes but when it is a XL model, I need first to load the XL base model. Nevermind if it is not in your roadmap, I have a hack.
